install-dependencies.sh

```diff
@@ -67,17 +80,13 @@ curl -1sLf "https://raw.githubusercontent.com/vectorizedio/seastar/master/instal
 
 set -e
 
-root="$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)"
-
-cd $root
-
-rm -rf $root/depot_tools
+mkdir -p /opt/
+rm -rf /opt/depot_tools
```

Comment:
Should this file use `$DEPOT_TOOLS_DIR`?

Reply:
oh, right. gonna change

Reply:
done